src/components/card/card.js

```diff
@@ -1,45 +1,36 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Icon from '@mapbox/mr-ui/icon';
+import IconText from '@mapbox/mr-ui/icon-text';
+
 import LevelIndicator from '../level-indicator/level-indicator';
 
 class Card extends React.PureComponent {
   render() {
     const { thumbnail, level, language, description, path, title } = this.props;
-    let renderedThumbnail = '';
-    let renderedLevel = '';
-    let renderedLanguage = '';
-    if (thumbnail) {
-      renderedThumbnail = <div className="relative h120 mb6">{thumbnail}</div>;
-    }
-    if (level) {
-      renderedLevel = (
-        <div className="inline-block mr18 txt-s txt-bold">
-          <LevelIndicator level={level} />
-        </div>
-      );
-    }
-    if (language) {
-      renderedLanguage = (
-        <div className="inline-block color-gray txt-s txt-bold">
-          <Icon name="code" inline={true} /> {language}
-        </div>
-      );
-    }
+    const renderedThumbnail = thumbnail && (
+      <div className="relative h120 mb6">{thumbnail}</div>
+    );
+    const renderedLevel = level && (
+      <div className="mr12">
+        <LevelIndicator level={level} />
+      </div>
+    );
+    const renderedLanguage = language && (
+      <IconText iconBefore="code">{language}</IconText>
+    );
+
     return (
       <a
-        className="color-gray-dark transition color-blue-on-hover round clip inline-block w-full unprose pb18"
+        className="dr-ui--card color-gray-dark transition color-blue-on-hover round clip inline-block w-full unprose pb18"
```

Comment:
💡 Fulfilling this small goal to add a self-identifying class name to each component https://github.com/mapbox/dr-ui/issues/199